### ISSUE TITLE
Install fails on first step of guide because of vite incompatibility

### ISSUE THREAD

**Issue by MSchmidt** [maglevhq/maglev-core#69]
I'm following the guide with:

```shell
rails new my-awesome-site \
  -m https://raw.githubusercontent.com/maglevhq/maglev-core/master/template.rb \
  --database=postgresql \
  --skip-action-cable
```

However, this fails with `vite-plugin-ruby@^3.2.0 might not be compatible with vite_ruby-3.5.0 (ArgumentError)`

**Comment by enriquesalceda** [comment]
@MSchmidt did you find a fix/work around?

**Comment by MSchmidt** [comment]
@enriquesalceda no I did only spent about 30 minutes on this before giving up and posting this. Was hoping for an upstream fix before getting back into it.

**Comment by ziraqyoung** [comment]
@enriquesalceda & @MSchmidt I was able to get around it by prefixing my commands with `VITE_RUBY_SKIP_COMPATIBILITY_CHECK=true` eg `VITE_RUBY_SKIP_COMPATIBILITY_CHECK=true bundle exec rails maglev:vite:install_dependencies` even for rails server. If you are desperate you can try that. 

I tried it on a new rails app instead with:
Ruby: 3.2.2
Rails: 7.1.2
Nodejs: 18:16.0

Without version pinning on Gemfile hence `maglevcms v1.4.0`

Check this: https://vite-ruby.netlify.app/config/#skipcompatibilitycheck

**Comment by did** [comment]
hey all, sorry for the delay. 
I'm working on it here 👉 https://github.com/maglevhq/maglev-core/pull/70
I hope to be done by the end of the week.

**Comment by natematykiewicz** [comment]
I see #70 was merged, would you be able to release a new version with this fix? I was trying Maglev out, but the quickstart guide fails to install because of this.

**Comment by did** [comment]
@natematykiewicz thanks for poking me on that! I've just released the 1.5.1 version.
